lib/active_admin/application.rb

```diff
@@ -250,6 +256,8 @@ def controllers_for_filters
       controllers
     end
 
+    attr_writer :routes_reloader # :nodoc:
```

Comment:
Don't intend this to a public API, but I'm unsure how you keep methods in the public namespace from showing up in documentation.